.github/workflows/luacheck.yml

```
@@ -40,7 +40,7 @@ jobs:
           luarocks install --deps-only luacheck-dev-1.rockspec
 
       - name: Build ‘luacheck’ (bootstrap)
-        run: luarocks make
+        run: luarocks make --pin
```

```suggestion
        run: luarocks make
```

This CI job can't make use of pinned dependencies because we test across an array of Lua versions.